site/content/blog/2020-10-01-whats-new-in-svelte-october-2020.md

```diff
@@ -0,0 +1,59 @@
+---
+title: What's new in Svelte: October 1, 2020
+description: New object methods, in-depth learning resources and tons of integration examples!
+author: Daniel Sandoval
+authorURL: https://desandoval.net
+---
+
+## New features
+1. `use:obj.method` allows functions defined within objects to be used within actions ([Example](https://svelte.dev/repl/c305722adb4a4545b27b198ea8ff9bde?version=3.26.0), **3.26.0**)
+2. `_` is now supported as a "numerical seperator", similar to a `.` or `,` ([Example](https://svelte.dev/repl/6612cec5c50b44cbab7f4a64ff930153?version=3.26.0), **3.26.0**)
+3. `import.meta` now works in template expressions ([Example](https://svelte.dev/repl/9630de41957a4c80a4fce264360a6bc7?version=3.26.0), **3.26.0**)
+
+These last two updates come from an update to Svelte’s upstream dependency, [acorn](https://github.com/acornjs/acorn). Acorn generates Svelte’s Abstract Syntax Tree (AST) and is explained in detail [here](https://github.com/sveltejs/svelte/issues/1011#issuecomment-351262252)!
+
+
```

Comment:
(cannot make this a suggestion because of the code block unfortunately):

4. When using TypeScript, you can now type the `createEventDispatcher` method:

```html
<script lang="ts">
    import { createEventDispatcher } from 'svelte';

    const dispatch = createEventDispatcher<{
        /**
         * you can also add docs
         */
        checked: boolean; // Will translate to `CustomEvent<boolean>`
        hello: string;
    }>();

    // ...
</script>
```
This will make sure that you can invoke `dispatch` only with the specified event names and its types. The `Svelte for VS Code` extension was also updated to deal with this new feature. It will provide autocompletion and hover information for these events and provide strong typing. Note though that this will _NOT_ make the events strict in the sense that you would get type errors when trying to listen to other events when using the component.